src/common/config/env.validation.ts

```diff
@@ -160,6 +160,7 @@ export class EnvironmentVariables {
   @IsNumber()
   @Min(74240) // Altair
   @Transform(({ value }) => parseInt(value, 10), { toClassOnly: true })
+  @ValidateIf((vars) => vars.NODE_ENV != Environment.test && vars.ETH_NETWORK === Network.Mainnet)
```

Comment:
I would suggest to have `network -> altair epoch` dict\enum. Check `START_EPOCH` by value and not check if the network isn't in the dict\enum

Reply:
Yes, I agree that the way that you suggested to do this is better. The problem here is that we need to validate the minimum value for the `START_EPOCH` variable depending on the value of `ETH_NETWORK` variable that we don't know at the compilation time. To make this possible we have to implement a custom validation directive.

This is a thing that is definitely worth doing, and I'm going to implement it later. For now, I just want to roll up a quick fix to allow this app to be deployed on the Holesky testnet faster.

I removed checking of the environment from my previous commit. So, now the `START_EPOCH` variable is validated in all environments and so, the validation becomes stronger.